.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.useBrowserPreview": true
```

Comment:
I assume we don't need this merged to main? We should probably remove this file from this PR.